internal/bits/max_amd64.s

```diff
@@ -458,7 +458,7 @@ TEXT ·maxBE128(SB), NOSPLIT, $-48
     CMPQ DX, $256
     JB loop
 
-    CMPQ ·hasAVX512MinMaxBE128(SB), $0
```

Comment:
Huh I'm not able to find much reading about the CMPQ instruction at all.

Reply:
https://www.felixcloutier.com/x86/cmp